apps/ledger-live-desktop/src/renderer/components/TransactionConfirm/index.tsx

```diff
@@ -129,14 +189,91 @@ const TransactionConfirm = ({ t, device, account, parentAccount, transaction, st
   const Warning = r?.warning;
   const Title = r?.title;
   const Footer = r?.footer;
-  const fields = getDeviceTransactionConfig({
-    account,
-    parentAccount,
-    transaction,
-    status,
-  });
+
   const key = ("mode" in transaction && transaction.mode) || "send";
   const recipientWording = t(`TransactionConfirm.recipientWording.${key}`);
+
+  console.log("typeTransaction", typeTransaction);
+
+  if (typeTransaction === "Approve" || typeTransaction === "Present")
```

Comment:
I don't think this will work as "Present" isn't a type of transaction afail.
It is set here https://github.com/LedgerHQ/ledger-live/blob/8fdc17621d6a5b238ae8517e490844af330a3f41/libs/coin-evm/src/deviceTransactionConfig.ts#L396 and the field is not `Type` but rather it is `Data`

Reply:
My bad just noticed you filtered correctly on `Data` fields also